### PR TITLE
Patch 3 - CalenStyle

### PR DESCRIPTION
**Pull request by nehakadam**
Created info.ini and update.json files inside newly created calenstyle folder.

### CODE REVIEW

File: files/calenstyle/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
```
Comment (jimaek) [inline files/calenstyle/update.json]: There are no tags here https://github.com/CuriousSolutions/CalenStyle/releases
The bot will fail

File: files/calenstyle/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "calenstyle",
+  "repo": "CuriousSolutions/CalenStyle",
+  "files": {
+    "basePath": "dist/",
+    "include": ["*"]
```
Comment (jimaek) [inline files/calenstyle/update.json]: Do you really need all those files?
How about using only the minified files?

File: files/calenstyle/update.json (hunk 1 above)
Comment (nehakadam) [inline files/calenstyle/update.json]: Added Releases.

**Comment by nehakadam** [comment]
Modified PR according to your suggestions.

File: files/calenstyle/update.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "packageManager": "github",
+  "name": "calenstyle",
+  "repo": "CuriousSolutions/CalenStyle",
+  "files": {
+    "basePath": "dist/",
+    "include": ["*.min.*", 
+                "calenstyle-jquery-ui-override.css", 
+                "calenstyle-iconfont.css", 
+                "fonts/calenstyle-iconfont.ttf",
```
Comment (jimaek) [inline files/calenstyle/update.json]: To retain the directory structure you need to add `./` in the beginning.
For example `./fonts/calenstyle-iconfont.ttf`
Please update both PRs. Thanks

**Comment by jimaek** [comment]
Ignore the bot :) Do it like I said in the comments

**Comment by nehakadam** [comment]
Ok. Thanks.

**Comment by nehakadam** [comment]
Do I have to rename calenstyle to jquery.calenstyle?

**Comment by jimaek** [comment]
If it requires jquery to work then yes

**Comment by nehakadam** [comment]
Done

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by nehakadam** [comment]
Thank You.

**Pull request closed without merging**